tests/shellcheck/run_shellcheck.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# If $top_srcdir has not been set by automake, import the test environment
+if [ -z "$top_srcdir" ]; then
+    top_srcdir="$(realpath "$(dirname "$0")/../..")"
+    . "${top_srcdir}"/tests/testenv.sh
```

Comment:
Do we really want to source this file? Isn't it going to affect the results of the check?

Reply:
I thought this is a standard thing to do? The other tests appear to work that way. That said, the file does not *seem* to contain anything relevant. Right now, at least.

Reply:
The file helps to overcome the fact that we run tests on the source code and not on the installed code, so the files can be at different locations than expected. I don't think that any linter cares about that. It doesn't look like we use this file for running `pylint`.

Reply:
You're right, that seems to be redundant. Removed.